core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/externaldatabase/ExternalDatabaseService.java

```diff
@@ -124,6 +133,20 @@ public void stopDatabase(Cluster cluster, DatabaseAvailabilityType externalDatab
         }
     }
 
+    private Optional<DatabaseServerV4Response> findExistingDatabase(Cluster cluster, String environmentCrn) {
+        Optional<DatabaseServerV4Response> existingDatabase = Optional.empty();
```

Comment:
drop this line

Reply:
if possible I like the single point of return, but, whatever. 🤷🏼‍♂️